src/main/java/com/example/bookstoreapp/config/SecurityConfig.java

```diff
@@ -30,13 +37,30 @@ public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Excepti
                 .csrf(AbstractHttpConfigurer::disable)
                 .authorizeHttpRequests(
                         auth -> auth
-                                .requestMatchers("/auth/**","/swagger-ui/**")
+                                .requestMatchers(
+                                        "/auth/**",
+                                        "/error",
+                                        "/swagger-ui/**",
+                                        "/v3/api-docs/**",
+                                        "/swagger-resources/**",
+                                        "/swagger-ui.html",
+                                        "/webjars/**")
```

Comment:
changes from previous PRS are still visible here

Reply:
> changes from previous PRS are still visible here

I did rebase.
It took quite long but those changes are still visible here
![image](https://github.com/fedaktaras/book-store-app/assets/24474626/e6b0d136-b673-4a82-812c-102c0818eee1)